docs/index.rst

```diff
@@ -1,14 +1,16 @@
 Welcome to MQT Bench's documentation!
 =====================================
 
-MQT Bench is a tool for benchmarking quantum software tools developed as part of the `Munich Quantum Toolkit <https://mqt.readthedocs.io>`_ (*MQT*) by the `Chair for Design Automation <https://www.cda.cit.tum.de/>`_ at the `Technical University of Munich <https://www.tum.de>`_.
+MQT Bench is a tool for benchmarking quantum software tools developed as part of the `Munich Quantum Toolkit <https://mqt.readthedocs.io>`_ (*MQT*) [^1].
 
 We recommend you to start with the :doc:`quickstart guide <Quickstart>`.
 If you are interested in the theory behind MQT Bench, have a look at the publication :cite:labelpar:`quetschlich2023mqtbench`.
 Furthermore, MQT Bench is also available as a `PennyLane dataset <https://pennylane.ai/datasets/single-dataset/mqt-bench>`_.
 
 We appreciate any feedback and contributions to the project. If you want to contribute, you can find more information in the :doc:`Contribution <Contributing>` guide. If you are having trouble with the installation or the usage of MQT Bench, please let us know at our :doc:`Support <Support>` page.
 
+[^1]: The _[Munich Quantum Toolkit (MQT)](https://mqt.readthedocs.io)_ is a collection of software tools for quantum computing developed by the [Chair for Design Automation](https://www.cda.cit.tum.de/) at the [Technical University of Munich](https://www.tum.de/) as well as the [Munich Quantum Software Company (MQSC)](https://munichquantum.software).
+
```

Comment:
RST uses a different syntax for links and footnotes, see https://docutils.sourceforge.io/docs/user/rst/quickref.html#footnotes

We shouldn't put too much effort into this fix here as the documentation will move to markdown anyway.